zerver/openapi/zulip.yaml

```diff
@@ -126,7 +126,7 @@ paths:
       security: []
       responses:
         "200":
-          description: Success.
+          description: A typical successful JSON response may look like
```

Comment:
Can we pull this description from a common entry rather than having N copies of it?

Reply:
That is what I was gonna do in the next commit :)(along with fixing the cases where the script didn't find any descriptions).